week-2/Homework/mandatory/4-sorting-algorithm.js

```diff
@@ -14,10 +14,35 @@ You don't have to worry about making this algorithm work fast! The idea is to ge
 "think" like a computer and practice your knowledge of basic JavaScript.
 */
 
-function sortAges(arr) {
+// function sortAges(arr) { - another way of only keeping numbers within a string
+//   let dataType;
+//   let newArray = [];
+//   for (let i = 0; i < arr.length; i++) {
+//     dataType = typeof (arr[i]);
+//     if (dataType === "number") {
+//       newArray.push(arr[i]);
+//     }
+//   }
 
+function sortAges(arr) { //new arr - typeof - push
+let newArr = [] // only keeps numbers within a string
+for (let index = 0; index < arr.length; index++) {
+  if (typeof arr[index] === "number" ) {
+    newArr.push(arr[index])
+  }
+}

```

Comment:
This is great! 

One thing that might make this better, especially with more complex functions in the future. Don't be afraid to extract small meaningful parts of the code as functions. You could create a nice function that filters the array. It makes the code easier to read and understand and allows you to later re-use the same part of code in other places.